Sources/SwiftDocC/DocumentationService/Models/Services/Convert/ConvertRequest.swift

```diff
@@ -130,6 +129,13 @@ public struct ConvertRequest: Codable {
     /// - ``DocumentationBundle/miscResourceURLs``
     public var miscResourceURLs: [URL]
     
+    /// The symbol identifiers that have an expanded documentation page available if they meet the associated access level requirement.
+    ///
+    /// DocC sets the ``RenderMetadata/hasExpandedDocumentationForSymbols`` property to `true`
+    /// for these symbols if their access level meets the minimum requirement, so that renderers can display a
+    /// "View More" link that navigates the user to the full version of the documentation page.
+    public var symbolIdentifiersWithExpandedDocumentation: [String: AccessControlLevel]?
```

Comment:
Should we consider passing a more generically named struct here that itself contains `AccessControlLevel`? I think it's likely we'll want to pass additional requirements in the future.

```suggestion
    public var symbolIdentifiersWithExpandedDocumentation: [String: ExpandedDocumentationRequirement]?
```

```swift
struct ExpandedDocumentationRequirement: Codable {
    let accessLevel: AccessControlLevel?
}
```